### PR TITLE
Feature/issue 1489 add logo to header menu

### PR DESCRIPTION
**Pull request by ian-lin-PG**

### CODE REVIEW

**Review by brandon-PG — APPROVED** [review]

File: src/application/images.ts [hunk 1]
```diff
@@ -20,3 +20,8 @@ export const emptySearch = require('../../assets/images/aa_empty_search.png');
 export const noMatchingSearchResults = require('../../assets/images/aa_no_matching_search_results.png');
 export const invalidSearchLocation = require('../../assets/images/aa_invalid_search_location.png');
 export const emptyBookmarks = require('../../assets/images/aa_empty_bookmarks.png');
+export const mb211Logo = require('../../assets/images/mb_211_logo.png');
+export const mbStartLogo = require('../../assets/images/mb_start_logo.png');
+export const pgLogo = require('../../assets/images/pg_logo.png');
```
Comment (rasmus-storjohann-PG) [inline src/application/images.ts:25]: peaceGeeksLogo, avoid abbreviations. MB is OK, but PG is not clear.

File: src/components/header_menu/header_menu_component.tsx [hunk 2]
```diff
@@ -208,6 +212,33 @@ const AboutItem = (props: { readonly icon: JSX.Element, readonly text: JSX.Eleme
...
+    } else {
+        return (
+            <View style={{ flexDirection: 'row', marginHorizontal: 10 }}>
+                <Image source={pgLogo} style={{ height: 40, width: 65 }} />
+                <Image source={welcomeBCLogo} style={{ height: 40, width: 95, marginHorizontal: 16 }} />
+                <Image source={bc211Logo} style={{ height: 40, width: 61 }} />
+            </View>
+        );
+    }
+};
```
Comment (rasmus-storjohann-PG) [inline src/components/header_menu/header_menu_component.tsx:240]: @brandon-PG you may be able to reuse this component.

**Review by rasmus-storjohann-PG — CHANGES_REQUESTED** [review]
Just minor changes suggested

File: src/application/images.ts [hunk 3]
```diff
@@ -24,3 +24,4 @@ export const mb211Logo = require('../../assets/images/mb_211_logo.png');
 export const mbStartLogo = require('../../assets/images/mb_start_logo.png');
 export const bc211Logo = require('../../assets/images/bc_211_logo.png');
 export const welcomeBCLogo = require('../../assets/images/welcome_bc_logo.png');
+export const pgLogo = require('../../assets/images/pg_logo.png');
```
Comment (rasmus-storjohann-PG) [inline src/application/images.ts]: commit message does not match code change

**Review by rasmus-storjohann-PG — CHANGES_REQUESTED** [review]
Just one commit message that needs to be fixed

**Pull request merged**